app.rb

```diff
@@ -1,2 +1,20 @@
 require 'sinatra'
-require 'sinatra/reloader'
\ No newline at end of file
+require 'sinatra/reloader'
+
+# get '/:num_bottles' do
+#   "99 Bottles of beer on the wall. <a href=/'98'>Take one down</a>"
+# end
+get '/' do
+  @bottles =
+  erb(:index)
```

Comment:
When we instantiate variables, we always need to assign it a variable when we use equal signs.